src/server.h

```diff
@@ -3197,6 +3198,13 @@ void hashTypeAddToExpires(redisDb *db, sds key, robj *hashObj, uint64_t expireTi
 void hashTypeFree(robj *o);
 int hashTypeIsExpired(const robj *o, uint64_t expireAt);
 unsigned char *hashTypeListpackGetLp(robj *o);
+size_t hashTypeListpackMemUsage(robj *o);
+int hashTypeListpackIsExpired(uint64_t expireTime);
```

Comment:
If I'm not mistaken, these functions declarations are letfovers.

Reply:
these two are new added, i guess you keep them when deal with conflict.

Reply:
yes, I had these in my PR. Then, I removed them. So, leftover declarations from the previous rebase.

Reply:
@tezc i saw you as Ronen.

Reply:
@sundb lol wish there is a way to change the avatar :D